main.cpp

```diff
@@ -1,53 +1,115 @@
-#include <SFML/Graphics.hpp>
```

Comment:
Aceeasi problema cu line ending-urile, practic o sa le rezolvi cu pasii dinainte.

Functia pe care am facut-o eu ai redenumit-o, nu chiar corect. Las-o loadBoard, pentru ca ea doar cu desenatul se ocupa, nu cu jocul in sine.

In functiile tale, recomand sa modifici si sa-ti faci variabilele camelCase. De exemplu, in loc de Scoretext, care-i greu de citit, incepi cu litera mica, pentru ca nu-s clase, si cand e cuvant nou pui litera mare. Basically Scoretext -> scoreText.

De asemenea, fisierele din folderele bin, debug si orice dll-uri n-au ce cauta pe git. Get 'em down.